stone/backends/python_client.py

```diff
@@ -163,6 +161,9 @@ def _generate_routes(self, namespace):
         Generates Python methods that correspond to routes in the namespace.
         """
 
+        # Hack: needed for _docf()
+        self.cur_namespace = namespace
+
```

Comment:
Random curiosity, what's the reason for moving this into `_generate_routes()`?

Reply:
Thanks for the quick approval! The tests call `_generate_routes` and `cur_namespace` needs to be set for `_docf` to work properly.